content/docs/security/backend-tls.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Backend TLS
+weight: 10
+description: Configure TLS to terminate for a specific backend workload. 
+---
+
+When you configure an [HTTPS listener](/docs/setup/listeners/https), the Gateway terminates the TLS connection and decrypts the traffic. The Gateway then routes the decrypted traffic to the backend service.
+
+However, you might have a specific backend workload that uses its own TLS certificate. In this case, you can configure the Gateway to originate a TLS connection that terminates at the backend service by using the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} BackendTLSPolicy. For more information, see the [{{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} docs](https://gateway-api.sigs.k8s.io/api-types/backendtlspolicy/).
+
+## Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+
+## Create a backend workload with a TLS certificate {#workload-tls-cert}
+
+The following example uses an NGINX server with a self-signed TLS certificate. For the configuration, see the [test directory in the {{< reuse "docs/snippets/product-name.md" >}} GitHub repository](https://github.com/kgateway-dev/kgateway/tree/main/test/kubernetes/e2e/features/backendtls/inputs).
+
+
+1. Deploy the NGINX server with a self-signed TLS certificate.
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/kgateway-dev/kgateway/refs/heads/main/test/kubernetes/e2e/features/backendtls/inputs/nginx.yaml
```

Comment:
general q - is `main` the branch we point to now for yaml links like this bc we are building/running off `-main`? Should we eventually make the branch name a conref to change the branch with each released version?

Reply:
I added a version conref for the github branch. The Hextra theme doesn't support versioning of the docs, so it will be interesting to see how we approach that down the road, not sure.